### ISSUE TITLE
Error During Make, Pkg Installed Anyway?

### ISSUE THREAD

**Issue by bhreach** [dslackw/slpkg#84]
export EXPERIMENTAL=YES
slpkg -s sbo f3

Error:

`cc -o f3fix libutils.o f3fix.o -lparted
f3fix.o: In function `argp_usage':
/usr/include/argp.h:564: multiple definition of `argp_usage'
libutils.o:/usr/include/argp.h:564: first defined here
f3fix.o: In function `_option_is_short':
/usr/include/argp.h:570: multiple definition of `_option_is_short'
libutils.o:/usr/include/argp.h:570: first defined here
f3fix.o: In function `_option_is_end':
/usr/include/argp.h:582: multiple definition of `_option_is_end'
libutils.o:/usr/include/argp.h:582: first defined here
collect2: error: ld returned 1 exit status
Makefile:38: recipe for target 'f3fix' failed
make: *** [f3fix] Error 1
make: *** Waiting for unfinished jobs....`

Package still installed?

**Comment by dslackw** [comment]
Work for me, but if you want to pass variable to script use export F3_EXPERIMENTAL=YES

**Comment by bhreach** [comment]
I found the problem. Comment from AltraMayor:

"I found a workaround that is being used by some packages while libargp is not patched: using option -fgnu89-inline. In order to compile F3 with -O2 (or any other optimization option), edit the Makefile as shown:

CFLAGS += -O2 -fgnu89-inline -std=c99 -Wall -Wextra -pedantic -MMD -ggdb

... The final solution is to wait for libargp to be patched."

Adding -fgnu89-inline to CFLAGS works for me.
